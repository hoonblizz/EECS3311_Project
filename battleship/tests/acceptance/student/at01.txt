-- custom_setup test for error cases
custom_setup_test(3,1,1,1)
custom_setup_test(4,8,1,1)
custom_setup_test(5,2,145,1)
custom_setup_test(5,2,3,8)