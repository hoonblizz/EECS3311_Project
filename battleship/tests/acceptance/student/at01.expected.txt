Type Error: specification of command executions is not type-correct
custom_setup_test(3, 1, 1, 1) does not conform to declaration custom_setup_test(dimension: GRID_SIZE = 4 .. 12 ; ships: NUMBER_OF_SHIPS = 1 .. 7 ; max_shots: MAX_SHOTS = 1 .. 144 ; num_bombs: NUMBER_OF_BOMBS = 1 .. 7)
custom_setup_test(4, 8, 1, 1) does not conform to declaration custom_setup_test(dimension: GRID_SIZE = 4 .. 12 ; ships: NUMBER_OF_SHIPS = 1 .. 7 ; max_shots: MAX_SHOTS = 1 .. 144 ; num_bombs: NUMBER_OF_BOMBS = 1 .. 7)
custom_setup_test(5, 2, 145, 1) does not conform to declaration custom_setup_test(dimension: GRID_SIZE = 4 .. 12 ; ships: NUMBER_OF_SHIPS = 1 .. 7 ; max_shots: MAX_SHOTS = 1 .. 144 ; num_bombs: NUMBER_OF_BOMBS = 1 .. 7)
custom_setup_test(5, 2, 3, 8) does not conform to declaration custom_setup_test(dimension: GRID_SIZE = 4 .. 12 ; ships: NUMBER_OF_SHIPS = 1 .. 7 ; max_shots: MAX_SHOTS = 1 .. 144 ; num_bombs: NUMBER_OF_BOMBS = 1 .. 7)
